service/src/battery.cpp

```diff
@@ -213,10 +256,58 @@ void Battery::showLowNotification() {
     }
 }
 
+void Battery::showHealthNotification() {
+    // set up alert categories
+    // TODO: manage this more globally, use better data types(?), align with QML/Settings part
+    static const QMap<QString, int> HealthThresh {
+        { "ok"   , 0},
+        { "warn" , 1},
+        { "crit" , 2},
+    };
+    // map string values from sysfs file to alert category
+    static const QMap<QString, int> HealthState {
+        { "good"     , HealthThresh["ok"] },
+        { "warm"     , HealthThresh["warn"] },
+        { "cool"     , HealthThresh["warn"] },
+        { "overheat" , HealthThresh["crit"] },
+        { "cold"     , HealthThresh["crit"] }
+    };
```

Comment:
This looks good... It's going to get a bit cumbersome combining two worded values, no matter how it's done.